enterprise/suppress-user/factory.go

```diff
@@ -123,6 +128,7 @@ func (m *Factory) Setup(ctx context.Context, backendConfig backendconfig.Backend
 			})
```

Comment:
maybe another `stopped` log like you've added for `latestRepo.Stop()` above?
Also not being fully aware, I hope there's distinction between `...sync stopped` and `...sync done` and that they're used appropriately

Reply:
`fullRepo` will be stopped only during shutdown of rudder-server. So, I don't think would be of much help. And, the reason we added log for `latestRepo` is because we are swapping `latestRepo` with `fullRepo` once `latestRepo` has all the suppression regulations.

Reply:
yes, `sync done` is when we are done with syncing with `suppression-backup-service`. And, `stop` is more like closing or getting rid of it.

Reply:
fair enough.
>fullRepo will be stopped only during shutdown of rudder-server

In any case, it'd be nice to have one imo.